server/tests/integration/user.spec.js

```diff
@@ -464,81 +463,14 @@ describe('User Api', function() {
         updated: user.updated
       }
 
-      await userReq.put('/api/users').send(requestBody).expect(200)
+      await userReq.put('/api/users/me').send(requestBody).expect(200)
 
       const updatedUser = await User.findById(user._id).lean()
       expect(updatedUser).to.have.property('email', requestBody.email)
       expect(updatedUser).to.have.property('firstName', requestBody.firstName)
       expect(updatedUser).to.have.property('lastName', requestBody.lastName)
     })
 
-    xit('regular users cannot update another user profile', async function(){
```

Comment:
I think some of these tests, and in the next file, could be useful to keep, maybe it's obvious from the code now that these issuees can't happen but it would be good to have some assurance that they'll never happen.

Reply:
Sure, I'll leave in some as regression tests. Should I add a comment referencing the bug in the code? I would normally just go ahead with it, but the contributors guidelines here are a little restrictive in how issue references can be used.

Reply:
It's up to you. I'm not sure why the guidelines say not to reference the issue in the PR.